lib/service/kubernetes.go

```diff
@@ -81,6 +81,12 @@ func (process *TeleportProcess) initKubernetesService(log *logrus.Entry, conn *C
 		return trace.Wrap(err)
 	}
 
+	// Start uploader that will scan a path on disk and upload completed
+	// sessions to the Auth Server.
+	if err := process.initUploaderService(accessPoint, conn.Client); err != nil {
```

Comment:
Has this been moved here from somewhere or was it just missing and it was an error/omission?

Reply:
It was missing here due to omission